packages/inter-protocol/src/proposals/price-feed-proposal.js

```diff
@@ -131,29 +132,36 @@ export const createPriceFeed = async (
 
   void E(client).assignBundle([_addr => ({ priceAuthority })]);
 
-  const timer = await chainTimerService;
+  let installationP;
+  await null;
+  if (priceAggregatorRef) {
+    const bundleID = await E.get(priceAggregatorRef).bundleID;
+    if (bundleID) {
+      installationP = E(zoe).installBundleID(bundleID);
```

Comment:
do we want to update `agoricNames.installation.priceAggregator` in this case?

Reply:
I think so, but i'm going to hold off until I see how we entrain the upgrades to auctions and vaults. If those are planned to follow shortly,  then yes, but if not, it depends on the reasoning.

Reply:
I mis-read your question. The answer to your actual question is yes, and we do so on line 141 now.

The answer to the question I thought you asked is "yes, we want to update the instances, and we were already doing so on line 222.